docs/use/kingfisher-process.rst

```diff
@@ -0,0 +1,19 @@
+Kingfisher Process
+==================
+
+Read the `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/>`__ documentation, which cover general usage.
+
+View Logs
+---------
+
+The logs from Process are available for the ``ocdskfs`` and ``ocdskfp`` users to read.
+
+The newest logs are at ``/var/log/kingfisher.log``.
+
+Log rotation is used, and older logs can be found at ``/var/log/kingfisher.log.1`` and other compressed files like ``/var/log/kingfisher.log.2.gz``.
+
+Logs are tagged with a logger, and you can use this to filter for events you want to see.
```

Comment:
Logs -> Log messages

I think it'd also be helpful to describe the full format of a log message, so that people know which parts aren't relevant to them: `[date] [hostname] "%(asctime)s - %(process)d - %(name)s - %(levelname)s - %(message)s"`. We can link to this page for folks to look up the meaning of each: https://docs.python.org/3/library/logging.html#logrecord-attributes.